tests/functional/u/useless/useless_with_lock.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=missing-docstring
```

Comment:
```suggestion
"""Tests for the useless-with-lock message"""
# pylint: disable=missing-docstring
```

I like adding docstrings at module level to describe what a test is supposed to test. Helps when we revisit them years later.
Will need to update the tests after this because of the added line.

Reply:
Added